app/controllers/menus_controller.rb

```diff
@@ -0,0 +1,6 @@
+class MenusController < ApplicationController
+  def show
+    @menus = Menu.where("restaurant_id = ?", params[:restaurant_id].to_i)
```

Comment:
What @tochman means here is that there's a cleaner way to do this. Have a look at the ActiveRecord documentation and see how to use the `where` method. Or look for any other method that does this.